pkg/deviceshifu/deviceshifulwm2m/deviceshifulwm2mconfig_test.go

```diff
@@ -0,0 +1,54 @@
+package deviceshifulwm2m
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLwM2MInstructions(t *testing.T) {
+	tests := []struct {
+		name           string
+		input          *deviceshifubase.DeviceShifuInstructions
+		expected       *LwM2MInstruction
+		expectingPanic bool
```

Comment:
when will except panic? It seems that you never use this field or expect is true